src/player.cpp

```diff
@@ -3837,7 +3837,7 @@ void player::set_cat_level_rec(const std::string &sMut)
     if (!has_base_trait(sMut)) { //Skip base traits
         const auto &mdata = mutation_branch::get( sMut );
         for( auto &elem : mdata.category ) {
-            mutation_category_level[elem] += 8;
```

Comment:
I figured having a more 1-based scale would be an improvement to the legibility of the code, but I've reverted that, since you seem to disagree. I haven't changed the display of the debug command, so that it represents the game's internal state literally.